packages/accounts/src/msca/account/multisigAccount.ts

```diff
@@ -0,0 +1,119 @@
+import type { EntryPointDef } from "@alchemy/aa-core";
+import {
+  createBundlerClient,
+  getAccountAddress,
+  getVersion060EntryPoint,
+  toSmartContractAccount,
+  type Address,
+  type SmartAccountSigner,
+  type SmartContractAccountWithSigner,
+  type UserOperationRequest,
+} from "@alchemy/aa-core";
+import {
+  concatHex,
+  encodeFunctionData,
+  hexToBigInt,
+  type Chain,
+  type Hex,
+  type Transport,
+} from "viem";
+import { MultisigModularAccountFactoryAbi } from "../abis/MultisigModularAccountFactory.js";
+import { multisigMessageSigner } from "../plugins/multisig/signer.js";
+import { getDefaultMultisigModularAccountFactoryAddress } from "../utils.js";
+import { standardExecutor } from "./standardExecutor.js";
+
+export type MultisigModularAccount<
+  TSigner extends SmartAccountSigner = SmartAccountSigner
+> = SmartContractAccountWithSigner<"MultisigModularAccount", TSigner>;
+
+export type CreateMultisigModularAccountParams<
+  TTransport extends Transport = Transport,
+  TSigner extends SmartAccountSigner = SmartAccountSigner
+> = {
+  transport: TTransport;
+  chain: Chain;
+  signer: TSigner;
+  salt?: bigint;
+  factoryAddress?: Address;
+  threshold: bigint;
+  sigs?: Address[];
+  entryPoint?: EntryPointDef<UserOperationRequest>;
+  accountAddress?: Address;
+  initCode?: Hex;
+};
+
+export async function createMultisigModularAccount<
+  TTransport extends Transport = Transport,
+  TSigner extends SmartAccountSigner = SmartAccountSigner
+>(
+  config: CreateMultisigModularAccountParams<TTransport, TSigner>
+): Promise<MultisigModularAccount<TSigner>>;
+
+export async function createMultisigModularAccount({
+  transport,
+  chain,
+  signer,
+  accountAddress,
+  initCode,
+  entryPoint = getVersion060EntryPoint(chain),
+  factoryAddress = getDefaultMultisigModularAccountFactoryAddress(chain),
+  sigs = [],
+  salt = 0n,
+  threshold,
+}: CreateMultisigModularAccountParams): Promise<MultisigModularAccount> {
+  const client = createBundlerClient({
+    transport,
+    chain,
+  });
+
+  const getAccountInitCode = async () => {
+    if (initCode) {
```

Comment:
nit:
```suggestion
    if (initCode != null) {
```

Reply:
ill have to change the one in multiowner also to be consistent, is that fine?

Reply:
maybe follow up with MO changes outside of this pr, think it's fine to have this do it this way but you're right it should be consistent

Reply:
I think it's ok to do an implied null check here since the type of `initCode` is `Hex?` which is only falsey if it's not present